README.md

```diff
@@ -24,3 +24,34 @@ translators in seconds.
 Zanata is Free software, licensed under the [LGPL][].
 
 [LGPL]: http://www.gnu.org/licenses/lgpl-2.1.html
+
+Developers
+----------
+
+*Prerequisites*
+
+You will need:
+- Java SDK 8 (OpenJDK recommended)
+- npm
+- Mysql or MariaDB
+- JBoss EAP 6 or Wildfly
+
+*Scripts*
+
+You can use one of the scripts to build the project using maven:
+
+[`etc/scripts/quickbuild.sh`](etc/scripts/quickbuild.sh) - Builds the project
+as quickly as possible, targeting both Firefox and Chrome when building GWT
+components, and skipping all checks and verifications (i.e. tests, checkstyle, etc)
+
+If you wish to build GWT components for chrome or firefox only, you can specify the
+`-c` and `-f` arguments respectively.
+
+The `-h` argument prints the script's help.
+
+[`etc/scripts/cargowait.sh`](etc/scripts/cargowait.sh) - Builds the Zanata artifact
+and starts a JBoss server using the cargo plugin. This script is particularly
+useful for starting a Zanata instance with the aim of running functional tests
+from an IDE.
```

Comment:
Would be good to have people able to skim this and find what they're looking for. `quickbuild.sh` gives a decent idea, but `cargowait.sh` is anyone's guess if they haven't used cargo before.  Maybe we could add a heading for each script that is descriptive of what it does.

e.g.

```md
### Quickly build to a .war file

[`etc/scripts/quickbuild.sh`](etc/scripts/quickbuild.sh) - Builds the project...
...

### Build and run server for testing

[`etc/scripts/cargowait.sh`](etc/scripts/cargowait.sh) - Builds the...
...
```

(does `quickbuild` generate a .war file?)